### ISSUE TITLE
PE machine types list missing AXP64

### ISSUE THREAD

**Issue by Wack0** [GerbilSoft/rom-properties#387]
COFF/PE machine type 0x0284 is AXP64, see the recently dumped 2210.main for this architecture: https://archive.org/details/axp64-2210-installable

This currently shows as unknown CPU type.

**Comment by GerbilSoft** [comment]
Please check this again with rom-properties v2.2. I fixed this in #380, and just verified it again with the AXP64 Windows build and it detects it correctly:

```
== Reading file './2210/axp64chk/pro/axp64/autofmt.exe'...
-- Microsoft Windows Executable detected
----- PE -----
Type:            '64-bit Portable Executable'
CPU:             'DEC Alpha AXP (64-bit)'
OS Version:      '5.0'
Subsystem:       'Native 5.0'
PE Flags:        [*] Executable [*] >2GB addressing [ ] DLL
DLL Flags:       [ ] High Entropy VA    [ ] Dynamic Base [ ] Force Integrity
                 [ ] NX Compatible      [ ] No Isolation [ ] No SEH         
                 [ ] No Bind            [ ] AppContainer [ ] WDM Driver     
                 [ ] Control Flow Guard [ ] TS Aware    
Timestamp:       03/03/2000 07:38:26 AM
```

**Comment by Wack0** [comment]
Ah, I checked the code before submitting this and didn't see it, sorry.
